colin/core/loader.py

```diff
@@ -124,6 +124,24 @@ def obtain_check_classes(self):
                         load_check_classes_from_file(path)))
         return list(check_classes)
 
+    def import_class(self, import_name):
+        """
+        import selected class
+
+        :param import_name, str, e.g. some.module.MyClass
+        :return the class
+        """
+        if six.PY3:
```

Comment:
Probably unnecessary. From the last release, we do not support Python2.

Reply:
nice! shall I remove all the python2 junk from this file?

Reply:
A hope so.